book/src/terminology.md

```diff
@@ -178,6 +182,13 @@ A set of [transactions](#transaction) that may be executed in parallel.
 
 The role of a [fullnode](#fullnode) when it is validating the [leader's](#leader) latest [entries](#entry).
 
+#### vote
+
+A validator's affirmation that the PoH it has received has been verified.  A vote contains a hash of the updated state since the previous [voting period](#voting period).
+
+#### voting period
+
+The length of time in [ticks](#tick), between each [vote](#vote) submitted by a [validator](#validator).  Also called [slot](#slot).
```

Comment:
Do we only allow one vote per slot?  If slots are measured in ticks and a tick is about a second, how do we ensure subsecond finality?

Reply:
if we separate voting and leader rotation period, we can vote as quickly as we like.  ticks are 100ms right now
https://github.com/solana-labs/solana/blob/777a0a858e59b3a3d8e7aa269d1a8941ce06c59b/src/poh_service.rs#L12